packages/marketplace/contracts/transfer-manager/TransferManager.sol

```diff
@@ -14,11 +15,15 @@ import {LibPart} from "../lib-part/LibPart.sol";
 /// @notice responsible for transferring all Assets
 /// @dev this manager supports different types of fees
 /// @dev also it supports different beneficiaries
-abstract contract TransferManager is ERC165Upgradeable, ITransferManager {
+abstract contract TransferManager is ERC165Upgradeable, AccessControlUpgradeable, ITransferManager {
     using BpLibrary for uint;
 
     bytes4 internal constant INTERFACE_ID_IROYALTYUGC = 0xa30b4db9;
 
+    /// @notice role to identify the sandbox accounts
+    /// @return hash for TSB_WALLET
+    bytes32 public constant TSB_WALLET = keccak256("TSB_WALLET");
```

Comment:
lets be specific but generic in the naming, my suggestion
`SKIP_FEES_ROLE`

(and suffix with _ROLE)